config/default/setup.cfg

```diff
@@ -3,6 +3,8 @@ universal = 1
 
 [flake8]
 doctests = 1
+# provided to doctests by buildoutSetUp()
+builtins = write, system, cat, join
```

Comment:
Should this be in `default/setup.cfg`?

Very few of ZF packages are buildout recipes that use doctests with `buildoutSetUp()`.

Reply:
According to https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations the configuration of `flake8` can be put into `tox.ini`. So I'd like to put it there as well as the `doctests = 1`, so the whole configuration is in one place. (I'd even change the other templates.) Would this be okay for you?

Reply:
I don't much care where the flake8 configuration lives.  I care about arbitrary getting names excluded from flake8 checks for packages that are not buildout recipes.

Reply:
I moved it to `tox.ini`, so the builtins are only defined for buildout-recipes.